app/components/avo/fields/textarea_field/show_component.html.erb

```diff
@@ -1,3 +1,3 @@
 <%= field_wrapper **field_wrapper_args do %>
-  <%= @field.value %>
+  <div style="white-space: pre-line;"><%= @field.value %></div>
```

Comment:
```suggestion
  <div class="whitespace-pre-line"><%= @field.value %></div>
```

Reply:
IDK why I can't also commit this in the UI.
Can you please commit this @krystof-k?
It's good for merge after that.